src/shared/components/adminTabs/Games/GamesManagement.tsx

```diff
@@ -53,11 +53,40 @@ function GamesManagement() {
         })])
     }
 
+    function areDistinct(arr)
+    {
+        let n = arr.length;
+    
+        let s = new Set();
+        for (let i = 0; i < n; i++) {
```

Comment:
Here we can use `arr.forEach` instead of doing a for like this. Let's use js methods as much as possible, because the code will be more clean and easier to read.